### PR TITLE
IGAPP-844: Wait until loading disappears

### PR DESCRIPTION
**Pull request by Taggotty**
This pull request belongs to an issue on our [bugtracker](https://issues.integreat-app.de/).
You can find it there by looking for an issue with the key which is mentioned in the title of this pull request.
It starts with the keyword **IGAPP**.

### CODE REVIEW

File: e2e-tests/native/wdio-browserstack.conf.ts [hunk 1]
```diff
@@ -30,17 +30,27 @@ export const config: Testrunner = {
...
   reporters: ['junit'],
 
   jasmineOpts: {
     defaultTimeoutInterval: 300000
+  },
+
+  before: async (): Promise<void> => {
+    // implicit: time the driver will wait searching for elements ($, $$)
+    const implicitTimeout = 2000
+    driver.setImplicitTimeout(implicitTimeout)
```
Comment (klinzo) [inline e2e-tests/native/wdio-browserstack.conf.ts:54]: Is this for all queries?

File: e2e-tests/native/wdio-browserstack.conf.ts [hunk 2]
```diff
@@ -30,17 +30,27 @@ export const config: Testrunner = {
 
   logLevel: 'info',
 
-  bail: 1,
   baseUrl: 'http://localhost:9000',
+
+  specFileRetries: 2,
```
Comment (klinzo) [inline e2e-tests/native/wdio-browserstack.conf.ts:35]: Does this retry the tests?

**Review by klinzo — APPROVED** [review]

File: e2e-tests/native/wdio-browserstack.conf.ts (hunk 1 above)
Comment (Taggotty) [reply e2e-tests/native/wdio-browserstack.conf.ts:54]: Sí.

File: e2e-tests/native/wdio-browserstack.conf.ts (hunk 2 above)
Comment (Taggotty) [reply e2e-tests/native/wdio-browserstack.conf.ts:35]: yes.

**Review by f1sh1918 — APPROVED** [review]

**Pull request merged**